php-plugin/pom.xml

```diff
@@ -16,21 +16,30 @@
     <url>https://github.com/green-code-initiative/ecoCode/tree/main/php-plugin</url>
 
     <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>ecocode-rules-specifications</artifactId>
+            <version>${project.version}</version>
+            <classifier>php</classifier>
+        </dependency>
 
         <dependency>
             <groupId>org.sonarsource.php</groupId>
             <artifactId>sonar-php-plugin</artifactId>
             <type>sonar-plugin</type>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>org.sonarsource.sonarqube</groupId>
             <artifactId>sonar-plugin-api</artifactId>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>org.sonarsource.sonarqube</groupId>
             <artifactId>sonar-plugin-api-impl</artifactId>
+            <scope>test</scope>
```

Comment:
why test scope ? did you test php plugin with real project ?

Reply:
I have tested plugin with SonarQube LTS and https://github.com/green-code-initiative/ecoCode-php-test-project.git